ServantHDBC/src/Api/Photo.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Api.Photo (PhotoAPI, photoHandlers) where
+
+import Control.Monad.Trans.Except
+import Servant
+
+type PhotoAPI = "photos" :>
+                (Capture "path_segment_1" String :>
+                 Capture "path_segment_2" String :>
+                 Capture "geometry_or_style" String :>
+                 Capture "original_filename" String :>
+                    Get '[JSON] String)
```

Comment:
what does `'[JSON]` here signify - the incoming content-type or the outgoing content-type?